src/genie/libs/parser/iosxe/show_interface.py

```diff
@@ -513,9 +510,8 @@ def cli(self,interface="",output=None):
                     interface_dict[interface]\
                                 ['oper_status'] = line_protocol
 
-                if line_attribute:
-                    interface_dict[interface]['connected'] = True if line_attribute == 'connected' else False
-                    interface_dict[interface]['err_disabled'] = True if line_attribute == 'err-disabled' else False
```

Comment:
Is there a reason the check for err-disabled was removed?

Reply:
I can't explain that, it shouldn't have been removed (i didn't actively remove it

Reply:
Code is there now. No issue, resolved